### PR TITLE
Add contentContainerStyle to ModalProps interface

### LINKED ISSUE #1051: Add contentContainerStyle property to ModalProps interface
ModalProps interface in Modal.d.ts is missing contentContainerStyle property from documentation

```
import * as React from 'react';
import { ThemeShape } from '../types';

export interface ModalProps {
  dismissable?: boolean;
  onDismiss?: () => any;
  visible: boolean;
  children: React.ReactNode;
  theme?: ThemeShape;
}

export declare class Modal extends React.Component<ModalProps> {}
```

### PR DESCRIPTION
**Pull request by Taym95**
Fix #1051

### CODE REVIEW

File: typings/components/Modal.d.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ export interface ModalProps {
   onDismiss?: () => any;
   visible: boolean;
   children: React.ReactNode;
+  contentContainerStyle?: any;
```
Comment (satya164) [inline typings/components/Modal.d.ts]: Can you change it to `StyleProp<ViewStyle>`?

Comment (Taym95) [reply typings/components/Modal.d.ts]: Yes

Comment (satya164) [reply typings/components/Modal.d.ts]: You need to import the types from `react-native`

Comment (Taym95) [reply typings/components/Modal.d.ts]: Oops, My bad sorry!

**Comment by vko-online** [comment]
not published yet :(

**Pull request merged**